proxmoxtf/resource/vm.go

```diff
@@ -3137,15 +3145,17 @@ func vmGetHostPCIDeviceObjects(d *schema.ResourceData) vms.CustomPCIDevices {
 		)
 		romfile, _ := block[mkResourceVirtualEnvironmentVMHostPCIDeviceROMFile].(string)
 		xvga := types.CustomBool(block[mkResourceVirtualEnvironmentVMHostPCIDeviceXVGA].(bool))
+		mapping, _ := block[mkResourceVirtualEnvironmentVMHostPCIDeviceMapping].(string)
 
 		device := vms.CustomPCIDevice{
-			DeviceIDs:  strings.Split(ids, ";"),
 			PCIExpress: &pcie,
 			ROMBAR:     &rombar,
 			XVGA:       &xvga,
+			Mapping:    &mapping,
```

Comment:
`Mapping` is assigned below on line 3170, so it can be removed from here as well.